contentcuration/contentcuration/frontend/channelEdit/components/AssessmentItemEditor/AssessmentItemEditor.vue

```diff
@@ -285,13 +285,6 @@
       if (!this.question) {
         this.openQuestion();
       }
-      // Assessments are nested inside of a scrolling panel.
-      // Instead of propagating an event all the way back to
-      // the scrolling panel, just use scrollIntoView
-      // (supported by most major browsers)
-      if (this.$el.scrollIntoView) {
```

Comment:
Hi @Jaspreet-singh-1032, thank you for the changes here. Although I have some queries, removing this if statement would completely eliminate the smooth animation effect. Instead, we may want to enable scrolling with animation to the "response type" field rather than the "title/question". @MisRob @LianaHarris360 can you guide on the exact behaviour here? Removing the if statement does seem to work altho there won't be any scroll animation.

Reply:
Yes, I think we would want to keep the animation scroll. As for enabling scrolling to the "response type" field, the `scrollIntoView()` method takes in a `block` argument in addition to `behavior`, that [defines the vertical alignment](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView#:~:text=of%20scroll%2Dbehavior-,block,-Optional) and the default is set to 'start' which doesn't seem to scroll to the preferred view, so possibly trying 'center' or 'nearest' could be an option.

Reply:
The maximum of the `response type` shown by using `block: end or nearest`.
This is how it looks.
![image](https://github.com/learningequality/studio/assets/69707565/e17e9a23-64f6-4a87-a6d3-19c167e16b50)

Reply:
It seems to me that it's caused by the secondary bar (with "Details", "Questions", "Related"). Even though the scroll position is correct, the element we scroll to is hidden underneath the bar. I think that one day to deal with it might be something like:
1. Get coordinates of the question type element
2. Scroll window (https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo) to those coordinates but with an offset that corresponds to the bar height.

Reply:
Thanks, I will try this solution. However, the `window.scrollTo` function was not working here, but I will research it. It could be due to some CSS.

Reply:
Ah, yes, this is a modal actually, so I assume that's the reason. Perhaps `Element.scrollTo()` https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTo could be used then.

Reply:
The `scrollTo` did not work on any of the elements. I think this is because there is an outer scrollable div in which all these elements are rendered and the `scrollTo` only works on the outer parent div and not on the internal elements.

Reply:
I was able to scroll using the outer div with `scrollTo` function. I will try to implement a workaround with this.

Reply:
Ah, that sounds tricky. Wonderful you found a workaround, let's see. Let us know if you needed anything.